bigtable/api/Snippets/Filters/FilterComposingChain.cs

```diff
@@ -0,0 +1,41 @@
+﻿
+// Copyright (c) 2020 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+// [START bigtable_filters_composing_chain]
+
+using Google.Cloud.Bigtable.V2;
+using System.Threading.Tasks;
+
+namespace Filters
+{
+    public class FilterComposingChain
+    {
+        /// <summary>
+        /// Read using a chain filter from an existing table.
+        ///</summary>
+        /// <param name="projectId">Your Google Cloud Project ID.</param>
+        /// <param name="instanceId">Your Google Cloud Bigtable Instance ID.</param>
+        /// <param name="tableId">Your Google Cloud Bigtable table ID.</param>
+        public static Task<string> BigtableFilterComposingChain(
+            string projectId = "YOUR-PROJECT-ID",
+            string instanceId = "YOUR-INSTANCE-ID", string tableId = "YOUR-TABLE-ID")
+        {
+            // A filter that selects one cell per column AND within the column family cell_plan
+            RowFilter filter = RowFilters.Chain(RowFilters.CellsPerColumnLimit(1), RowFilters.FamilyNameExact("cell_plan"));
+            return ReadFilter.BigtableReadFilter(projectId, instanceId, tableId, filter);
```

Comment:
All of these read filter samples are on the same page, and we realized that having the repeated reading with filter code included in each was very distracting because the goal is to show constructing the filter. We would still include the code that shows how to read with a filter at the top of the page and say something like "Create your filter here"